authors/apps/highlights/views.py

```diff
@@ -0,0 +1,66 @@
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.exceptions import ValidationError
+from rest_framework import status
+from .utils import HighlightUtils
+
+from ..articles.models import ArticleModel
+from .serializers import HighlightsSerializer
+from .models import HighlightsModel
+
+
+class HighlightArticleView(GenericAPIView):
+    """Highlight an article view"""
+    permission_classes = [IsAuthenticated]
+    serializer_class = HighlightsSerializer
+
+    def post(self, request, *args, **kwargs):
+        """Highlight an article"""
+
+        slug = self.kwargs.get('slug')
+        article = ArticleModel.objects.all().filter(slug=slug).first()
+        user = request.user
+        highlight = request.data.get("highlight", "")
+        location = request.data.get("location")
+
+        if HighlightUtils().validate_location(location):
```

Comment:
Great implementation in checking for the location of the highlighted message. Otherwise LGTM.